internal/pkg/fmt/fmt_test.go

```diff
@@ -13,50 +13,64 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 limitations under the License.
 */
-package fmt
+package fmt_test
```

Comment:
Why do you change name of this package? Isn't it standard way to create test files in same package as tested functions.

Reply:
I'm in favor of the style described e.g. in this [great post](https://medium.com/@matryer/5-simple-tips-and-tricks-for-writing-unit-tests-in-golang-619653f90742#:~:text=Put%20your%20tests%20in%20a,real%20user%20of%20the%20package.)